product_docs/docs/biganimal/release/using_cluster/05b_major_version_upgrade.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Upgrading Postgres major versions
+navTitle: Upgrading Postgres major versions
+---
+
+To upgrade small to moderately large databases (up to several tens of gigabytes) between major versions of Postgres on BigAnimal, we recommend utilizing pg_dump to migrate data and populate a new BigAnimal instance/cluster. This method will require some downtime, so schedule the pg_dump operation accordingly.
+
+## Migrating Postgres Versions with pg_dump
+
+!!! Important
+This procedure will likely accrue ingress and egress costs from your cloud service provider (CSP) for the migrated data. Please consult your CSP's pricing documentation to see how ingress and egress fees are calculated to determine any extra costs.
```

Comment:
I think we can link to CSP's pricing documentation from here if that is available.

```suggestion
This procedure accrues ingress and egress costs from your cloud service provider (CSP) for the migrated data. Please consult your CSP's pricing documentation to see how ingress and egress fees are calculated to determine any extra costs.
```